### DISCUSSION TITLE
下载过滤器使用不了

### DISCUSSION THREAD

**Discussion by Ninethnine** [hect0x7/JMComic-Crawler-Python#260] [Q&A]
按照文档中的示例编程，

无法通过
![image](https://github.com/user-attachments/assets/005d7bd3-85a7-4dd1-9a24-5b1366adddec)

**Comment by hect0x7** [comment]
你会python吗？

**Reply by Ninethnine** [reply]
不会，但是我严格按照文档来的

**Reply by Ninethnine** [reply]
哦多打了个括号

**Reply by hect0x7** [reply]
你的用法不对，文档里有这行代码吗？
![image](https://github.com/user-attachments/assets/86c6a9eb-c9e1-42f8-b827-e69f05daf018)

**Reply by Ninethnine** [reply]
我的问题，谢谢解答，已经解决了
